modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminNavigationDisplayContext.java

```diff
@@ -131,16 +131,16 @@ public JSONArray getKBFolderDataJSONArray() throws PortalException {
 			));
 	}
 
-	public long getKBObjectToMoveParentId() throws PortalException {
-		String kbObjectToMoveClassName = ParamUtil.getString(
-			_httpServletRequest, "kbObjectToMoveClassName");
+	public long getMoveKbObjectParentId() throws PortalException {
+		String moveKbObjectClassName = ParamUtil.getString(
```

Comment:
the "b" should be uppercase: `getMoveKBObjectParentId()`, `moveKBObjectClassName`